support-frontend/typescript-errors.json

```diff
@@ -2,25 +2,25 @@
     "errorCounts": {
         "byCode": {
```

Comment:
Not too sure why this file has jumped around so much but a scan of the errors by code will show that the codes that have changed have all seen a decrease in number